spec/features/3d_viewer_spec.rb

```diff
@@ -15,6 +15,6 @@
   end
 
   it 'has working panels' do
-    expect(page).to have_content('About this item')
+    expect(page).to have_selector('h3', text: 'About this item', visible: :all)
```

Comment:
Should we click the button to open the panel?

Reply:
I think this is a good idea. It allows for us to make sure it isn't expanding when it shouldn't be